### ISSUE TITLE
Graphical Representation of H5YR tags usage (Chart/Map)

### ISSUE THREAD

**Issue by sumeshkp18** [H5YR/H5YR#16]
In the dashboard can we use Graphical representations like [https://www.highcharts.com](https://www.highcharts.com)
So we can be show the Linear growth of usage, the users from different countries used the tags in Map charts.
Once we fetch the data like this we can also generates report which country/area/community used most times etc.

**Comment by sumeshkp18** [comment]
Yes. I mean in the public web as well as the back end dashboard.

Web page view would be very helpful to understand and get overview for the end users.
Backend Dashboard view would be helpful in both very easy to get overview and also will help to generate the reports in quickly. 
Based on the report, admins can plan many features like Alex's idea Kudos Prize Draw etc.

Does it make sense?

**Comment by OwainWilliams** [comment]
Ye, that does make sense.
